lib/utils.js

```diff
@@ -160,6 +161,37 @@ function splitStringIntoChunks(inputString, chunkSize) {
   return inputString.match(regex);
 }
 
+const bytesToHexString = (bytes) => {
+  return bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
+};
+
+const removeEmptyLeftBytes = (storageValue) => {
+  const result = `${storageValue.replaceAll(/^0x0+/g, '')}`;
+  if(!result.startsWith('0x')) {
+    return `0x${result}`;
+  }
+  return result;
+};
+
+const decodeRlp = (rlpEncoded) => {
+  const uint8ArrayDecoded = RLP.decode(rlpEncoded);
+  const bytesStr = bytesToHexString(uint8ArrayDecoded);
+  return bytesStr;
+};
+
+const getBridgeStorageValueDecodedHexString = (bridgeStorageValueEncodedAsRlp, append0xPrefix = true) => {
```

Comment:
I think we implemented this in one of our libraries didn't we? Might be a good idea anyway to have maybe a specialized library that reads values from the Bridge storage